packages/snaps-sdk/src/jsx/components/Tooltip.ts

```diff
@@ -0,0 +1,44 @@
+import { createSnapComponent } from '../component';
+import type { StandardFormattingElement } from './formatting';
+import type { ImageElement } from './Image';
+import type { LinkElement } from './Link';
+import type { TextElement } from './Text';
+
+export type TooltipChildren =
+  | TextElement
+  | StandardFormattingElement
+  | LinkElement
+  | ImageElement
+  | null;
+
+/**
+ * The props of the {@link Tooltip} component.
+ *
+ * @property children - The children of the box.
+ */
+export type TooltipProps = {
+  children: TooltipChildren;
+  value: TextElement | StandardFormattingElement | LinkElement | string;
```

Comment:
```suggestion
  label: TextElement | StandardFormattingElement | LinkElement | string;
```
Potentially a better name? WDYT?

Reply:
I'm not sure `label` is the correct term to define the content of a Tooltip 🤔

Reply:
`content` could work too 🤔 

E.g. Chakra uses `label` and NextUI uses `content`. Just trying to pick something that devs might be used to

Reply:
I'm okay with content :)